### ISSUE TITLE
`undefined` on X509Certificate

### ISSUE THREAD

**Issue by duane-unitive** [tngan/express-saml2#18]
I'm constructing an IDP with settings data. The resulting metadata includes the signing certificate data as expected, but the string `undefined` is appended to the end. This seems incorrect.

```
<X509Certificate>...
Tfq+Hb+5g/UZsWfMY/czR+0mYawjundefined</X509Certificate>
```

What am I doing wrong?

**Comment by duane-unitive** [comment]
There is a bug in Utility.js:

`replace('-----END CERTIFICATE-----');`
should be
`replace('-----END CERTIFICATE-----', '')`;

**Comment by tngan** [comment]
Merged pull request #19
